packages/react-storage/src/components/StorageBrowser/do-not-import-from-here/types.ts

```diff
@@ -20,6 +20,7 @@ export interface LocationConfig {
   accountId?: string;
   bucket: string;
   credentialsProvider: LocationCredentialsProvider;
+  customEndpoint?: string;
```

Comment:
nit: is the type from here still used somewhere in ur PR?

Reply:
Good question, yep,
`LocationConfig` is referenced in `TaskActionInput`, `ListActionInput` and `DownloadActionInput` types on this file. It's used on all the `do-not-import-from-here/actions/*`files ([example](https://github.com/aws-amplify/amplify-ui/pull/5985/files#diff-d90dfff69123a63827049eccb3911e149f1592a2ce02a775450de4bb9618b0c5R37))

Currently we introduced the `customEndpoint?` type in two places `StorageBrowser/do-not-import-from-here/actions/` and `StorageBrowser/actions/handlers/*`. Some views use the former while others use the latter.

We're in the process of migrating all former ones to latter hence introduced the type in both places to be on the safer side